pkg/utils/finalizer_utils_test.go

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright (c) 2018 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package utils
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+func TestFinalizer(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Zookeeper Finalizers")
+}
+
+var _ = Describe("Zookeeper Finalizers", func() {
+
+	Context("creating strSlice", func() {
+		var val1, val2 bool
+		BeforeEach(func() {
+			var strSlice = []string{"10", "20"}
+			val1 = ContainsString(strSlice, "10")
+			val2 = ContainsString(strSlice, "30")
+		})
+		It("should return true for value 10", func() {
+			Ω(val1).To(Equal(true))
+		})
+		It("should return false for value 30", func() {
+			Ω(val2).To(Equal(false))
+		})
+	})
+
+	Context("creating strSlice", func() {
+		var result []string
+		BeforeEach(func() {
+			var strSlice = []string{"10", "20"}
+			result = RemoveString(strSlice, "10")
+		})
+		It("should return false for value 10", func() {
+			Ω(ContainsString(result, "10")).To(Equal(false))
+		})
+	})
+	Context("creating strSlice", func() {
```

Comment:
change context description to IsPvCOrphan

Reply:
Done